Users/abehenderson/CSX/Callbacks/intersection.js

```diff
@@ -0,0 +1,50 @@
+function intersection(arrays) {
+  
+  let result = arrays.reduce((acc, array) => {
+
+    const cache = [];
+
+    for (let i = 0; i < acc.length; i++) {
+      if (array.includes(acc[i])) cache.push(acc[i]);
+    }
```

Comment:
You _could_ simplify this by using `forEach`, but yours is fine.  
```suggestion
   acc.forEach(e => {
      if (array.includes(e)) cache.push(e);
    })
```

Reply:
There's a perception that normal "for" loops shouldn't be used in production code; being specific with your type of loop is a demonstration that you understand it well enough to be specific. Functionally there's nothing wrong with your code.

Reply:
Ok, this is good.  I've been practicing more with `forEach` and getting more comfortable with it.  Initially, I was most comfortable with `for...in` loops.  I've noticed that it takes longer for me to understand and be comfortable with methods, because I can't see what is happening under the hood.